orchestrator/cli/generator/generator/migration.py

```diff
@@ -12,21 +12,28 @@
 # limitations under the License.
 import re
 import subprocess  # noqa: S404
+from os import environ
+from pathlib import Path
 from typing import Generator, Optional
 
 import structlog
 
-from orchestrator.cli.generator.generator.helpers import get_variable
-
 logger = structlog.getLogger(__name__)
 
 
 def create_migration_file(config: dict) -> Optional[str]:
-    create_migration_command = f'PYTHONPATH=. DATABASE_URI=postgresql://nwa:nwa@localhost/nwa-workflows python main.py db revision --message "Add {config["name"]}" --head=data@head'
+    if not environ.get("DATABASE_URI"):
+        environ.update({"DATABASE_URI": "postgresql://nwa:nwa@localhost/orchestrator-core"})
+    if not environ.get("PYTHONPATH"):
+        environ.update({"PYTHONPATH": "."})
+    logger.info(
+        "creating new db revision", database_uri=environ.get("DATABASE_URI"), pythonpath=environ.get("PYTHONPATH")
+    )
+    create_migration_command = f'python main.py db revision --message "Add {config["name"]}" --head=data@head'
     result = subprocess.check_output(create_migration_command, shell=True, text=True)  # noqa: S602
 
     for line in result.splitlines():
-        if m := re.search("migrations/versions/general/([^ ]+)", line):
+        if m := re.search("migrations/versions/schema/([^ ]+)", line):
```

Comment:
Hmm, in surf orchestrator the path is `migrations/versions/general/`, in core it is indeed `migrations/versions/schema/` .. I think we want the former ? 🤔 

I think we've had a discussion about making this configurable, not sure though. Maybe @tjeerddie remembers?

Reply:
Not sure why different version locations are being used, @tjeerddie do you know why this is?

I can add a field to the product yaml to indicate the version location. I would prefer to keep the default  migrations/versions/schema/ as this is what most user use.